libqfieldsync/utils/qgis.py

```diff
@@ -42,7 +42,7 @@ def open_project(filename: str, filename_to_read: Optional[str] = None) -> bool:
     project = QgsProject.instance()
     project.clear()
 
-    is_success = project.read(filename_to_read or filename)
+    is_success = project.read(filename or filename_to_read)
```

Comment:
hm, are you sure here? This change will always read the filename, as it is a mandatory non-empty parameter.

Reply:
Yes, by doing this little change the sync work and open the project, if not inform `The data has been synchronized successfully but the original project home/seqlaz/QField/export/test_001.qgz could not be opened`

Reply:
Actually, by doing more tests the first commit is not needed, but this change is needed to fix the issue.